config.ts

```diff
@@ -0,0 +1,8 @@
+export interface Config {
+  service?: string;
```

Comment:
Use a service context schema similar to this [definition](https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/blob/20e69f23d3da2f405f40cbe16df606159d199fa0/config.ts#L125) from trace agent (debug agent [definition](https://github.com/GoogleCloudPlatform/cloud-debug-nodejs/blob/015f29c842111b96f6db062adee070f7bce83a5c/src/agent/config.ts#L34)).

Side-bar: we will get these into a common module down the road.